pkgs/development/libraries/libvirt/default.nix

```diff
@@ -150,35 +156,32 @@ stdenv.mkDerivation rec {
   strictDeps = true;
 
   nativeBuildInputs = [
-    meson # needs to be first
```

Comment:
Should we keep this comment?

Reply:
It needed to be "first"(actually just above cmake) to prevent cmake settings in configurePhase first

A var can be used to disable the cmake configurePhase but cmake is unnecessary so I just removed it